verification/cbmc/proofs/aws_cryptosdk_serialize_frame/Makefile

```diff
@@ -31,8 +30,16 @@ PROJECT_SOURCES += $(SRCDIR)/source/framefmt.c
 
 PROOF_SOURCES += $(COMMON_PROOF_SOURCE)/make_common_data_structures.c
 PROOF_SOURCES += $(COMMON_PROOF_SOURCE)/proof_allocators.c
+PROOF_SOURCES += $(COMMON_PROOF_SOURCE)/utils.c
 PROOF_SOURCES += $(PROOFDIR)/$(HARNESS_FILE)
 PROOF_SOURCES += $(PROOF_SOURCE)/make_common_data_structures.c
-###########
+
+REMOVE_FUNCTION_BODY += aws_byte_cursor_advance
```

Comment:
This improves our coverage metric, by discarding functions that are unreachable for this proof. The else branch in `field_sized` is unreachable, so these functions are never called; however, our coverage analysis still considers this path in the coverage metric. By removing this unreachable functions, we improve coverage metric. If they ever become reachable, our report will through a warning.

Reply:
Of the function bodies removed here, it looks like only `aws_byte_cursor_advance` appears directly in the else branch in question. Is it sufficient to remove just that function body, or do we need to manually list all the other functions that are transitively called, as done here?

Reply:
@alex-chew we only need to specify the functions that are directly included in the path, you're correct. I updated the list of these function for the `aws_cryptosdk_serialize_frame` proof and add a comment explaining why we do this.

Reply:
`aws_byte_cursor_read_be32` and `aws_byte_cursor_read_be64` functions are included in the coverage metric because of `field_helper`, but we also never reach the else branch there either.